manifests/0000_50_cluster-monitoring-operator_04-config.yaml

```
@@ -994,6 +994,11 @@ data:
     #
     # cluster:vsphere_csi_migration:max shows if CSI migration for vSphere was enabled or not.
     - '{__name__="cluster:vsphere_csi_migration:max"}'
+    #
+    # owners: (@openshift/openshift-team-api, @polynomial)
+    #
+    # apiserver_list_watch_request_total:rate:sum represents the rate of change for the LIST and WATCH requests over a 5 minute period.
+    - '{__name__="apiserver_list_watch_request_total:rate:sum"}'
```

```suggestion
    - '{__name__="apiserver_list_watch_request_success_total:rate:sum", verb=~"LIST|WATCH"}'
```
This seems redundant right now, but we want to keep adding these additional cardinality protections to our telemetry rules. This way the recording rule can changed in the future to include other verbs, without immediately affecting telemetry.